xain_fl/grpc/coordinator.py

```diff
@@ -262,30 +275,42 @@ class Coordinator:
     # pylint: disable-msg=dangerous-default-value
     def __init__(
         self,
-        num_rounds: int = 10,
-        required_participants: int = 10,
-        aggregator: Optional[Aggregator] = None,
```

Comment:
because we have no way to set it from the CLI parameters, so for now we can just remove it. also, it's not clear if we should be the one making sure the best possible aggregating algorithm should be used (in that case it should not be configurable at all) or the user should have a say. what do you think?

Reply:
update: after talking to @finiteprods , i put `aggregator` back into the constructor because it's easier for testing (ie: easier to swap to a simpler algorithm), even if the CLI doesn't accept such argument

Reply:
an additional request as a follow-up: can you also do the same for `controller` as well?

Reply:
no probs!